apparelo/apparelo/doctype/lot_creation/lot_creation.py

```diff
@@ -202,11 +202,17 @@ def get_ipd_item(doc):
 	item_production_detail=doc.get('item_production_detail')
 	item_template=frappe.db.get_value("Item Production Detail", {
 	                                  'name': item_production_detail}, "item")
+	ipd_sizes =[]
+	for _size in frappe.get_doc('Item Production Detail',item_production_detail).size:
+		ipd_sizes.append(int(_size.size[:-3]))
```

Comment:
we can use the size list from IPD directly for ordering. we can assume that it is the correct for sizes.